pkg/events/derive/hooked_syscall.go

```diff
@@ -3,68 +3,69 @@ package derive
 import (
 	"fmt"
 
+	lru "github.com/hashicorp/golang-lru/v2"
+
 	"github.com/aquasecurity/libbpfgo/helpers"
 
 	"github.com/aquasecurity/tracee/pkg/errfmt"
 	"github.com/aquasecurity/tracee/pkg/events"
 	"github.com/aquasecurity/tracee/pkg/events/parse"
-	"github.com/aquasecurity/tracee/pkg/utils"
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
-var SyscallsToCheck = make([]string, 0)
-var MaxSupportedSyscallID = events.IoPgetevents // Was the last syscall introduced in the minimum version supported 4.18
+var (
+	reportedHookedSyscalls *lru.Cache[int32, uint64]
+)
+
+// InitHookedSyscall initialize lru
+func InitHookedSyscall() error {
+	var err error
+	reportedHookedSyscalls, err = lru.New[int32, uint64](700)
```

Comment:
this hard coded 700 should be a const at the top of the file and 700 in eBPF code your MAX_SYS_CALL_TABLE_SIZE Is 500. Perhaps they should match.